File: Examples/ProgrammersGuide/AsposeCells/WorkingWithFiles/FileHandlingFeatures/FileHandlingFeaturesExamples.cs
using Helpers;
using NUnit.Framework;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using Assert = NUnit.Framework.Assert;
using Description = Microsoft.VisualStudio.TestTools.UnitTesting.DescriptionAttribute;

namespace Examples.ProgrammersGuide.AsposeCells.WorkingWithFiles.FileHandlingFeatures
{
    [TestClass, TestFixture]
    public class FileHandlingFeaturesExamples
    {	
        [TestMethod, Test, Owner("Console")]
        public void OpeningFiles()
        {
            TestHelper.SetDataDir("ProgrammersGuide/AsposeCells/WorkingWithFiles/FileHandlingFeatures/OpeningFiles");

            OpeningFilesExample.Program.Main();
        }

        [TestMethod, Test, Owner("Console")]
        public void SaveWorkbookToTextCSVFormat()
        {
            TestHelper.SetDataDir("ProgrammersGuide/AsposeCells/WorkingWithFiles/FileHandlingFeatures/SaveWorkbookToTextCSVFormat");

            SaveWorkbookToTextCSVFormatExample.Program.Main();
        }

        [TestMethod, Test, Owner("Console")]
        public void SavingFiles()
        {
            TestHelper.SetDataDir("ProgrammersGuide/AsposeCells/WorkingWithFiles/FileHandlingFeatures/SavingFiles");

            SavingFilesExample.Program.Main();
        }

    }
}

File: Examples/ProgrammersGuide/AsposeCells/WorkingWithFiles/FileHandlingFeatures/FileHandlingFeaturesExamples.vb
Imports Helpers
Imports NUnit.Framework
Imports Microsoft.VisualStudio.TestTools.UnitTesting
Imports Assert = NUnit.Framework.Assert
Imports Description = Microsoft.VisualStudio.TestTools.UnitTesting.DescriptionAttribute

Namespace Examples.ProgrammersGuide.AsposeCells.WorkingWithFiles.FileHandlingFeatures
	<TestClass, TestFixture> _
	Public Class FileHandlingFeaturesExamples
        <TestMethod(), Test(), Owner("Console")> _
	        Public Sub OpeningFiles()
	            TestHelper.SetDataDir("ProgrammersGuide/AsposeCells/WorkingWithFiles/FileHandlingFeatures/OpeningFiles")
	
	            OpeningFilesExample.Program.Main()
        End Sub

        <TestMethod(), Test(), Owner("Console")> _
	        Public Sub SaveWorkbookToTextCSVFormat()
	            TestHelper.SetDataDir("ProgrammersGuide/AsposeCells/WorkingWithFiles/FileHandlingFeatures/SaveWorkbookToTextCSVFormat")
	
	            SaveWorkbookToTextCSVFormatExample.Program.Main()
        End Sub

        <TestMethod(), Test(), Owner("Console")> _
	        Public Sub SavingFiles()
	            TestHelper.SetDataDir("ProgrammersGuide/AsposeCells/WorkingWithFiles/FileHandlingFeatures/SavingFiles")
	
	            SavingFilesExample.Program.Main()
        End Sub

	End Class
End Namespace